packages/graalvm/src/main/kotlin/elide/runtime/intrinsics/js/FetchMutableRequest.kt

```diff
@@ -43,13 +37,6 @@ public interface FetchMutableRequest : FetchRequest {
    * TBD.
    */
   public companion object DefaultFactory : RequestFactory<FetchMutableRequest> {
-    /**
-     * TBD.
-     */
-    @JvmStatic override fun forRequest(request: java.net.http.HttpRequest): FetchMutableRequest {
-      return FetchRequestIntrinsic.forRequest(request)
-    }
```

Comment:
removing support for creating Fetch requests from `java.net.HttpRequest`; there are module path issues